chapter5/dynamic-form/test_round_trip_destination.py

```
@@ -0,0 +1,44 @@
+    ids=[
+        "simple",
+        "round-trip-with-de",
+        "round-trip-without-de",
+        "one-way-with-de",
+        "one-way-without-de",
+    ],
+)
+def test_offline_cli_extracts_only_the_destination(user_request, tmp_path):
+    """Trip-type modifiers must not become part of the submitted destination."""
+    demo = Path(__file__).with_name("demo.py")
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)